File: Minimum Number of Operations to Make Array XOR Equal to K.cpp
class Solution {
public:
    int minOperations(vector<int>& nums, int k) {
        for (int& num: nums) {
            k ^= num;
        }

        int count = 0;
        while (k > 0)  {
            count += k % 2;
            k >>= 1;
        }
        return count;
    }
};

File: Minimum Number of Operations to Make Array XOR Equal to K.py
class Solution:
    def minOperations(self, nums: List[int], k: int) -> int:
        n = len(nums)
        for i in range(n):
            k ^= nums[i]
        count = 0
        while k:
            count += k%2
            k >>= 1
        return count